skimage/measure/_regionprops.py

```diff
@@ -112,75 +81,69 @@ class _RegionProperties(object):
     def __init__(self, slice, label, label_image, intensity_image,
                  cache_active):
         self.label = label
+
         self._slice = slice
         self._label_image = label_image
         self._intensity_image = intensity_image
+
         self._cache_active = cache_active
+        self._cache = defaultdict(lambda: None)
```

Comment:
Whenever it accesses that key, if the key is not there it adds a key value pair anyway.

But no matter, I changed this in the latest update.